spec/requests/file_attachments_spec.rb

```diff
@@ -101,6 +101,18 @@
         .to redirect_to(file_attachment_path(edition.document, file_attachment))
     end
 
+    it "redirects to attachments index view when featured attachment is created successfully" do
```

Comment:
I think this test goes in a different commit, as it relates to the 'success' case.